client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -159,7 +161,7 @@ export function PlanningForBrowserSupport() {
       <p>
         Make sure that, before you begin to plan your support strategy, you are
         all talking about the same thing when you talk about{" "}
-        <strong>supporting a browser</strong>.
+        <strong>supporting a browser.</strong>
```

Comment:
Punctuation goes outside of styling, so the period shouldn't be bolded. I asked Ruth about this because it's on a case by case basis, and she says MDN will not style punctuation.